great-idea/services.html

```diff
@@ -4,103 +4,131 @@
 <head>
   <meta charset="utf-8">
 
-  <title>Great Idea!</title>
+  <title>Great Idea! - Services</title>
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/services.css">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
-<body>
+<body class="container">
 
 
-Services
-Product
-Vision
-Features
-About
-Contact
+  <div class="line1">
+    <nav class="buttons">
+      <a class="link" href="services.html" target=_blank>Services</a>
+      <a class="link" href="">Product</a>
+      <a class="link" href="">Vision</a>
+      <a class="link" href="">Features</a>
+      <a class="link" href="">About</a>
```

Comment:
Your code is very clean. I'm going to nitpick only because you're doing such a good job. I'd like to see some css hover and click animations on your links and buttons. Also, you should have a link back to your index.html from the services page. 

The stretch problems are optional, but I think you can handle it. There are usually awesome lessons buried within the stretch. Some of the coolest things I've learned were introduced to me in the stretch problems. Also, the stretch problems will often give you a preview of lessons that are down the road.